CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ endif()
 add_executable(lsd)
 
 file(GLOB_RECURSE sources src/*.cpp)
-target_sources(lsd PUBLIC ${sources} "${CMAKE_SOURCE_DIR}/dependencies/src/glad.c")
+target_sources(lsd PUBLIC ${sources} "${CMAKE_SOURCE_DIR}/dependencies/src/glad.c" "header/move_to_component.h" "src/MoveToComponent.cpp")
```

Comment:
Shouldn't need to add header files to sources.

Reply:
When I remove them:

![Screenshot 2024-05-16 104426](https://github.com/MeindertKempe/LSD/assets/101462876/d0edf812-1c96-456d-a403-ea952cd689c9)